AwsEncryptionSDK/runtimes/go/ImplementationFromDafny-go/go.mod

```diff
@@ -1,46 +1,46 @@
-module github.com/aws/aws-encryption-sdk
+module github.com/aws/aws-encryption-sdk-dafny/releases/go/encryption-sdk
```

Comment:
Wasn't the conclusion of the Go Module Naming meeting was to change the repo name?

Reply:
I was expecting `github.com/aws/aws-encryption-sdk/releases/go/encryption-sdk`

Reply:
You are right, we need to rename the repo. I'd merge this PR and then do the rename before the release.